app/controllers/v3/domains_controller.rb

```diff
@@ -187,6 +192,7 @@ def verify_shared_organizations_guids!(message, owning_org_guid)
     organizations.each do |org|
       unprocessable!("Organization with guid '#{org.guid}' either does not exist, or you do not have access to it.") unless permission_queryer.can_read_from_org?(org.guid)
       unprocessable!("You do not have sufficient permissions for organization '#{org.name}' to share domain.") unless permission_queryer.can_write_to_org?(org.guid)
+      unprocessable!("Organization '#{org.name}' is suspended.") unless permission_queryer.is_org_active?(org.guid)
```

Comment:
Add tests to `spec/request/domains_spec.rb`:
- POST /v3/domains [private domain]
- PATCH /v3/domains/:guid
- DELETE /v3/domains/:guid
- POST /v3/domains/:guid/relationships/shared_organizations
- DELETE /v3/domains/:guid/relationships/shared_organizations/:org_guid

Reply:
Done